src/pkg/reg/adapter/awsecr/adapter.go

```diff
@@ -67,7 +67,7 @@ func parseAccountRegion(url string) (string, string, error) {
 	if rs == nil {
 		return "", "", errors.New("bad aws url")
 	}
-	return rs[1], rs[2], nil
+	return rs[1], rs[3], nil
```

Comment:
Please add length check for rs before this line.

Reply:
Happy to add it, but shouldn't that always evaluate to true based on the docs for FindStringSubmatch? In the event of no match it would just be nil.